Week 2/Weekly Practice Problems (Week 2)/1688: Count of Matches in Tournament/agunn.java

```diff
@@ -0,0 +1,15 @@
+class Solution {
+    public int numberOfMatches(int n) {
+        int count = 0;
+        while (n > 0) {
```

Comment:
Because there needs to be at least 2 players for a match to happen, we can change the condition for our loop to be `n > 1` instead. 
```suggestion
        while (n > 1) {
```